src/SOS/Strike/util.cpp

```diff
@@ -716,6 +736,34 @@ void DisplayThreadStatic(DacpModuleData* pModule, DacpMethodTableData* pMT, Dacp
         {   
             CLRDATA_ADDRESS appDomainAddr = vThread.domain;
 
+            // Get the DLM (we need this to check the ClassInit flags).
+            // It's annoying that we have to issue one request for
+            // domain-neutral modules and domain-specific modules.
+            DacpDomainLocalModuleData vDomainLocalModule;                
+            if (fIsShared)
+            {
+                if (g_sos->GetDomainLocalModuleDataFromAppDomain(appDomainAddr, (int)dwModuleDomainID, &vDomainLocalModule) != S_OK)
+                {
+					// On .NET Core, dwModuleDomainID is the address of the DomainLocalModule.
+					if (vDomainLocalModule.Request(g_sos, dwModuleDomainID) != S_OK)
```

Comment:
FYI, this is one of the changes that fix the dumpobj statics bug.  The rest of the changes in util.cpp are the coreclr sync.